azure/ConsiderationsForServiceDesign.md

```diff
@@ -225,33 +226,106 @@ a _status monitor_, which is an ephemeral resource that will track the status an
 The status monitor resource is distinct from the target resource (if any) and specific to the individual
 operation request.
 
-A POST or DELETE operation returns a `202 Accepted` response with the status monitor in the response body.
-A long-running POST should not be used for resource create -- use PUT as described below.
-PATCH must never be used for long-running operations -- it should be reserved for simple resource updates.
-If a long-running update is required it should be implemented with POST.
+There are four types of LROs allowed in Azure REST APIs:
+
+1. An LRO to create or replace a resource that involves additional long-running processing.
+2. An LRO to delete a resource.
+3. An LRO to perform an action on or with an existing resource (or resource collection).
+4. An LRO to perform an action not related to an existing resource (or resource collection).
+
+The following sections describe these patterns in detail.
+
+### Create or replace a resource requiring additional long-running processing
+<a href="#put-with-additional-long-running-processing"></a> <!-- Preserve anchor of previous heading -->
+
+A special case of long-running operation that occurs often is a PUT operation to create or replace a resource
+that involves some additional long-running processing.
+One example is a resource that requires physical resources (e.g. servers) to be "provisioned" to make the resource functional.
+
+In this case:
+- The operation must use the PUT method (NOTE: PATCH is never allowed here)
+- The URL identifies resource being created or replaced.
+- The request and response body have identical schemas & represent the resource.
+- The request may contain an `Operation-Id` header that the service will use as
+the ID of the status monitor created for the operation.
```

Comment:
This won't render correctly. Either needs to be over 2 spaces, or just put it on the last line:

```suggestion
- The request may contain an `Operation-Id` header that the service will use as the ID of the status monitor created for the operation.
```